cmd/main.go

```diff
@@ -73,6 +78,24 @@ func main() {
 	logger := controller.NewProductionGRPCLogger(bufferedGrpcSyncer)
 	defer logger.Sync() //nolint:errcheck
 
+	// Convert zap.Logger to logr.Logger
+	logrLogger := zapr.NewLogger(logger)
+
+	// Set logrLogger as the global logger for klog
+	klog.SetLoggerWithOptions(logrLogger)
+
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		klog.Error(err, "Failed to load in-cluster config")
+		return
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		klog.Error(err, "Failed to create clientset")
+		return
+	}
+
```

Comment:
I suspect that you only added this code to test.
That's OK.
Please remove this code now.
```suggestion
```